### ISSUE TITLE
No support for CDL MVWs makes a less-than-friendly user experience for multi-reel microfilm ingest  

### ISSUE THREAD

**Issue by kelea99** [pulibrary/figgy#4316]
A troubleshoot would be to add structure to the batch ingest of the MVWs  as one work. However, we are currently unable to add basic structure (that will be preserved) to what otherwise would be MVWs in the form of Microfilm reels for CDL items.  We only have one use case that I am aware of right now, but feel this may be a growing issue for the future, in regards to microfilm. @roelmoon and @kelynch, anything to add? 

Posli︠e︡dnīi︠a︡ novosti [microform]: https://figgy.princeton.edu/catalog/71ee8016-609c-4b53-bfc0-fe2259f44904

**Comment by kelea99** [comment]
CDL now supports mvws. Closing issue.
